src/layout/Navigation.tsx

```diff
@@ -1,16 +1,41 @@
 import { Link } from 'react-router-dom'
 import { COMMON_ROUTES_NAMES } from '@/routing'
+import { logOut, useAuth } from '@/firebase'
+import { createNavMenu } from './helpers'
+import classes from './Header.module.scss'
```

Comment:
not sure if it is good that Header and Navigation styles are in the same file. create separate folders for the heade fand navigation and separate the styles as well